### PR TITLE
Additional moderation

### PR DESCRIPTION
**Pull request by estrattonbailey**
- [x] Update API endpoint
- [x] Test

### CODE REVIEW

File: src/state/geolocation.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+async function getGeolocation(): Promise<Device['geolocation']> {
+  const res = await fetch(`https://api.bsky.app/xrpc/_health`)
+
+  if (!res.ok) {
+    throw new Error(`geolocation: lookup failed ${res.status}`)
+  }
+
+  const json = {country_code: 'BR'} // await res.json()
```
Comment (tangzero) [inline src/state/geolocation.tsx:37]: ???

**Review by pfrazee — APPROVED** [review]
Good work. Reviewed code, didnt test

File: src/state/geolocation.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+/**
+ * Default geolocation value. IF undefined, we fail closed and apply all
+ * additional mod authorities.
+ */
+export const DEFAULT_GEOLOCATION: Device['geolocation'] = {
+  countryCode: undefined,
+}
+
+async function getGeolocation(): Promise<Device['geolocation']> {
+  const res = await fetch(`https://app.staging.bsky.dev/ipcc `)
```
Comment (surfdude29) [inline src/state/geolocation.tsx:32]: Sorry for jumping in here, just wondering, is there supposed to be a space at the end of the URL here? 🤔

File: src/screens/Moderation/index.tsx [hunk 3]
```diff
@@ -423,45 +424,52 @@ export function ModerationScreenInner({
...
+                            handle={labeler.creator.handle}
+                          />
+                        </LabelingService.Content>
+                      </LabelingService.Outer>
+                    )}
+                  </LabelingService.Link>
+                </React.Fragment>
+              )
+            })}
         </View>
```
Comment (coreh) [inline src/screens/Moderation/index.tsx:473]: A suggestion/question: Would it be possible to add a notice to the UI whenever a non-configurable authority is enabled for the current region?

image

```suggestion
        </View>
      {labelers?.some(
        labeler => !isNonConfigurableModerationAuthority(labeler.creator.did),
      ) && (
        <View
          style={[
            a.p_lg,
            a.rounded_sm,
            t.atoms.bg_contrast_25,
            a.mt_xl,
            a.flex_row,
            a.gap_md,
          ]}>
          <View
            style={[
              a.align_center,
              a.justify_start,
              a.p_md,
              {width: 40},
              // t.atoms.bg_contrast_200,
            ]}>
            <Flag style={t.atoms.text_contrast_medium} />
          </View>
          <Text
            style={[
              a.leading_snug,
              t.atoms.text_contrast_high,
              a.flex_shrink,
              a.pr_sm,
            ]}>
            <Trans>
              In addition to the moderation services configured above, takedown
              requests from local authorities of the current region (
              {device.get(['geolocation'])?.countryCode ?? 'Unknown'}) are
              enabled in compliance with local laws.
            </Trans>
          </Text>
        </View>
      )}
```

File: src/state/geolocation.tsx [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+/**
+ * Default geolocation value. IF undefined, we fail closed and apply all
+ * additional mod authorities.
+ */
+export const DEFAULT_GEOLOCATION: Device['geolocation'] = {
+  countryCode: undefined,
+}
+
+async function getGeolocation(): Promise<Device['geolocation']> {
+  const res = await fetch(`https://app.staging.bsky.dev/ipcc`)
```
Comment (gaearon) [inline src/state/geolocation.tsx:32]: will this change to prod later?

**Review by gaearon — APPROVED** [review]

File: src/screens/Moderation/index.tsx (hunk 3 above)
Comment (estrattonbailey) [reply src/screens/Moderation/index.tsx]: Thanks for this suggestion! We discussed internally and decided to show the regional labeler when its applied, even though it's non-configurable. That way it's very clear what's happening.

Comment (coreh) [reply src/screens/Moderation/index.tsx]: Awesome! Thanks for this. It strikes a healthy balance between following local laws and still providing transparency about them.

**Pull request merged**